src/fenced/fenced/fence.py

```diff
@@ -55,10 +56,27 @@ def load_disks(self):
         unsupported = []
         remote_keys = set()
 
+        disks = sysctl.filter('kern.disks')[0].value.split()
+
+        # Running fenced exluding all disks is not allowed
+        try:
+            if not len(set(self._exclude_disks) - set(disks)):
+                raise PanicExit('Excluding all disks is not allowed')
+        except PanicExit:
```

Comment:
Why have it in a try/except if we are catching PanicExit to just raise it again?

Reply:
don't like panic'ing in this scenario so I've added another exception.